pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/actions/RestartDeclarativePipelineAction.java

```diff
@@ -75,7 +77,7 @@ private RestartDeclarativePipelineAction(Run run) {
 
     @Override public String getIconFileName() {
         // TODO: Needs an icon.
```

Comment:
obsolete comment

Reply:
No, we're still waiting on the real one - this is just a placeholder.